src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -380,7 +392,7 @@ public CommandLineParser getCommandLineParser() {
             commandLineParser = useLegacyParser(getClass()) ?
                         new LegacyCommandLineArgumentParser(this) :
                         new CommandLineArgumentParser(this,
-                            Collections.EMPTY_LIST,
+                            Collections.emptyList(),
```

Comment:
If we end up keeping both this code and the new code above, it should be refactored (e.g. as a method that accepts `boolean useLegacyParser`) so the code is only written once.

Reply:
the plan is NOT to keep both parsers for long....